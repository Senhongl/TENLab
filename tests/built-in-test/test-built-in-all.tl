a = [0,1,2];
if (all(a)){
    print("all-built-in functions doesn't work");
}
b = [2,1,2];
if (all(b)){
    print("all-built-in functions works!");
}