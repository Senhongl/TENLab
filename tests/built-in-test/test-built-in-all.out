all-built-in functions works!